config/config.go

```diff
@@ -111,13 +112,28 @@ func getEtcdURL() (string, error) {
 	return e, nil
 }
 
-func getFrontendURL() (string, error) {
-	e := os.Getenv("FRONTEND_URL")
+// getCorsAllowedOriginList fetches CorsAllowedOriginList from the env named CORS_ALLOWED_ORIGIN_LIST.
+// This allowed list is applied to kube-apiserver and the simulator server.
+//
+// e.g. CORS_ALLOWED_ORIGIN_LIST="http://localhost:3000, http://localhost:3001, http://localhost:3002"
+//       → return []string{"http://localhost:3000", "http://localhost:3001", "http://localhost:3002"}
+func getCorsAllowedOriginList() ([]string, error) {
+	e := os.Getenv("CORS_ALLOWED_ORIGIN_LIST")
 	if e == "" {
-		return "", xerrors.Errorf("get FRONTEND_URL from env: %w", ErrEmptyEnv)
+		return nil, xerrors.Errorf("get CORS_ALLOWED_ORIGIN_LIST from env: %w", ErrEmptyEnv)
```

Comment:
Should I also raise an error if the URL is informal? (Or fix to return `""`?)

Reply:
Updated.